akka-remote/src/main/scala/akka/remote/artery/EnvelopeBufferPool.scala

```diff
@@ -357,6 +385,12 @@ private[remote] final class EnvelopeBuffer(val byteBuffer: ByteBuffer) {
 
   private var literalChars = new Array[Char](64)
   private var literalBytes = new Array[Byte](64)
+  private var _streamId: Int = -1
```

Comment:
This is a bit unclean, so I guess we should at least document here how it is used.